can/interfaces/pcan/pcan.py

```diff
@@ -521,6 +547,14 @@ def _detect_available_configs():
             )
         return channels
 
+    def allow_error_frames(self, allow=True):
```

Comment:
I think we can just enable error frames in the `__init__` function for the `PcanBus`, much like the `SocketcanBus` does.

Reply:
Ok will do. Thanks